core/src/main/scala/org/apache/spark/memory/StoreUnifiedManager.scala

```diff
@@ -18,136 +18,86 @@ package org.apache.spark.memory
+  def dropStorageMemoryForObject(objectName: String, memoryMode: MemoryMode,
+                                 ignoreNumBytes: Long): Long
 
-  def releaseStorageMemoryForObject(objectName : String, numBytes: Long,
+  def releaseStorageMemoryForObject(objectName: String, numBytes: Long,
                                     memoryMode: MemoryMode): Unit
 
   def getStoragePoolMemoryUsed(memoryMode: MemoryMode): Long
+
   def getStoragePoolSize(memoryMode: MemoryMode): Long
+
   def getExecutionPoolUsedMemory(memoryMode: MemoryMode): Long
+
   def getExecutionPoolSize(memoryMode: MemoryMode): Long
+
   def getOffHeapMemory(objectName: String): Long
+
   def hasOffHeap: Boolean
+
   def logStats(): Unit
+
   /**
-   * Change the off-heap owner to mark it being used for storage.
-   * Passing the owner as null allows moving ByteBuffers not allocated
-   * by [[BufferAllocator]]s to be also changed and freshly accounted.
-   */
+    * Change the off-heap owner to mark it being used for storage.
+    * Passing the owner as null allows moving ByteBuffers not allocated
+    * by [[BufferAllocator]]s to be also changed and freshly accounted.
+    */
   def changeOffHeapOwnerToStorage(buffer: ByteBuffer,
-      allowNonAllocator: Boolean): Unit
+                                  allowNonAllocator: Boolean): Unit
 }
 
 /**
-  * This class will store all the memory usage for GemFireXD boot up time when SparkEnv
-  * is not initialised.
-  * This class will not actually allocate any memory.
-  * It is just a temp account holder till SnappyUnifiedManager is started.
+  * A MemoryManager which simply delegates to configured Spark memory manager.
+  * This manager will be used in Connector mode. All SnappyData execution memory
+  * like encoder etc will account memory here.
   */
-class TempMemoryManager extends StoreUnifiedManager with Logging{
-
-  val memoryForObject = new mutable.HashMap[(String, MemoryMode), Long]()
+class DefaultMemoryManager extends StoreUnifiedManager with Logging {
 
   override def acquireStorageMemoryForObject(objectName: String,
-      blockId: BlockId,
-      numBytes: Long,
-      memoryMode: MemoryMode,
-      buffer: UMMMemoryTracker,
-      shouldEvict: Boolean): Boolean = synchronized {
-    val key = objectName -> memoryMode
-    logDebug(s"Acquiring mem [TEMP] for $key $numBytes")
-    if (!memoryForObject.contains(key)) {
-      memoryForObject.put(key, 0L)
-    }
-    memoryForObject(key) += numBytes
-    true
+                                             blockId: BlockId,
+                                             numBytes: Long,
+                                             memoryMode: MemoryMode,
+                                             buffer: UMMMemoryTracker,
+                                             shouldEvict: Boolean): Boolean = {
+    logDebug(s"Acquiring DefaultManager meemory for $objectName $numBytes")
+    SparkEnv.get.memoryManager.acquireStorageMemory(blockId, numBytes, memoryMode)
```

Comment:
Correct the formatting in this class. The previous indentation was correct. Perhaps you need to set the codeStyleSettings.xml again with "Project" type in IDEA.